### ISSUE TITLE
LICENSE file

### ISSUE THREAD

**Issue by ChristianDDinga** [ChristianDDinga/python-intermediate-inflammation#2]
add missing LICENSE file

**Comment by Stefanie-B** [comment]
@ChristianDDinga I agree that this is legally very important and that I need to reference you.

**Comment by ChristianDDinga** [comment]
Great! This also gives me the opportunity to not only mention you @Stefanie-B, but to also mention an issue #3
